src/styles/blocks/star.scss

```diff
@@ -0,0 +1,14 @@
+.star {
+  width: $star-width;
+  height: $star-height;
```

Comment:
The variable `$star-height` is used here. Ensure that it is defined in a separate SCSS file dedicated to variables, as per the task requirements.